chain-tx-enclave-next/enclave-ra/ra-client/src/verifier.rs

```diff
@@ -291,13 +291,19 @@ impl EnclaveCertVerifier {
     }
 
     /// Converts enclave certificate verifier into client config expected by `rustls`
-    pub fn into_client_config(self) -> Result<ClientConfig, EnclaveCertVerifierError> {
+    pub fn into_client_config(
+        self,
+        verify_mr_enclave: bool,
```

Comment:
Can we have an enum instead of a boolean flag, makes the code loads easier to read.